complaint_search/es_interface.py

```diff
@@ -298,3 +299,36 @@ def document(complaint_id):
     res = _get_es().search(index=_COMPLAINT_ES_INDEX,
                            doc_type=_COMPLAINT_DOC_TYPE, body=doc_query)
     return res
+
+
+def states_agg(agg_exclude=None, **kwargs):
+    params = copy.deepcopy(PARAMS)
+    params.update(**kwargs)
+    search_builder = SearchBuilder()
+    search_builder.add(**params)
+    body = search_builder.build()
+
+    log = logging.getLogger(__name__)
+    log.info(
+        'Calling %s/%s/%s/states with %s',
+        _ES_URL, _COMPLAINT_ES_INDEX, _COMPLAINT_DOC_TYPE, body
+    )
+
+    # AD, TODO: Do I need to hardcode?
+    body["size"] = 0
+    del body['_source']
+    del body['highlight']
+    del body['sort']
```

Comment:
Maybe you shouldn't be calling `SearchBuilder`?  Or change `StateAggregationBuilder` to build what you need

Reply:
Originally I had planned to refactor much of `es_interface` and `es_builder` to support exactly that (and minimize copied code). The only reason I skipped the refactor was due to time constraints and other priorities. In addition to what you mentioned, I'd also like to separate out `SearchBuilder` from `AggregationBuilder` like we do in V2. I think we'll come back to it once work begins on trends/compare/geo